src/styles/main.scss

```
@@ -12,3 +12,30 @@ body {
 
 @import "_header.scss";
 @import "_catalog.scss";
+
+.catalog {
```

```suggestion
.catalog {
```
В тебе вже є клас `.catalog` у файлі `_catalog.scss`. Чому б не прописати ці стилі там?

---

неуважність)